packages/cf-builder-pagination/src/PaginationBuilder.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Pagination, PaginationItem } from 'cf-component-pagination';
-import Icon from 'cf-component-icon';
+import Icon from '../../cf-component-icon/src';
```

Comment:
Hmm. I think I added this because CI was failing. I'll remove all these and make a commit with it and hopefully it will just work :)